tests/bluetooth/bsim_bt/bsim_test_audio/src/vcs_client_test.c

```diff
@@ -15,7 +15,6 @@
 #define VOCS_DESC_SIZE 64
 #define AICS_DESC_SIZE 64
 
-static struct bt_conn_cb conn_callbacks;
```

Comment:
Kind of disagree. As @Thalley  mentioned it has been already fixed in separate PR. After merging the separate PR this change will just disappear. In the case of a separate commit, it will require git magic to get rid of the useless commit.
I would suggest to wait for https://github.com/zephyrproject-rtos/zephyr/pull/38297 merge

Reply:
@alxelax Could you review #38297 so that it can be merged?

Reply:
(At that point I wasn't aware of Thalley's PR). Is there a PR also for the other fix? (otherwise it would be nicer to keep it as a separate commit)